lib/daru/view/adapters/googlecharts.rb

```diff
@@ -56,7 +56,11 @@ def init_script
         end
 
         def generate_body(plot)
-          plot.to_html
+          plot.to_html[0]
+        end
+
+        def generate_id(plot, id=nil)
```

Comment:
I  am not satisfied with this kind of approach. Anyways I don't understand why there is `id` parameter and function name `generate_id`.

Reply:
As the generated `id` is random, I have used the `id` parameter for the testing purpose [here](https://github.com/SciRuby/daru-view/pull/84/files#diff-80df96fa561900cfd2a2182046316859R146)